tests/chocolatey-tests/commands/choco-uninstall.Tests.ps1

```diff
@@ -181,15 +196,210 @@ Describe "choco uninstall" -Tag Chocolatey, UninstallCommand {
+        # We might want to rollback files in this case, but it is not possible as the backup has been removed before
+        # any locked files are being tried to be removed.
+        It "Should have removed <_>" -ForEach @('isdependency', 'hasdependency') -Tag FossOnly {
+            "$env:ChocolateyInstall\lib\$_" | Should -Not -Exist -Because $Output.String
+        }
+
+        It "Should not have removed isexactversiondependency" {
+            "$env:ChocolateyInstall\lib\isexactversiondependency" | Should -Exist -Because $Output.String
+        }
+
+        It "Outputs <_> was succcesfully uninstalled" -ForEach @('isdependency', 'hasdependency') -Tag FossOnly {
+            $Output.Lines | Should -Contain "$_ has been successfully uninstalled." -Because $Output.String
+        }
+
+        It "Does not output isexactversiondependency being uninstalled" {
+            $Output.Lines | Should -Not -Contain "isexactversiondependency has been successfully uninstalled." -Because $Output.String
+        }
+
+        It "Outputs 2/2 packages uninstalled" {
+            $Output.Lines | Should -Contain "Chocolatey uninstalled 2/2 packages." -Because $Output.String
+        }
+    }
+
+    Context "When specifying multiple packages where one is a dependency should not fail uninstallation (forced dependencies)" {
+        BeforeAll {
+            Restore-ChocolateyInstallSnapshot
+
+            $null = Invoke-Choco install hasdependency --confirm
+
+            $Output = Invoke-Choco uninstall isdependency hasdependency --confirm --force-dependencies
+        }
+
+        It "Exits with Success (0)" {
+            $Output.ExitCode | Should -Be 0 -Because $Output.String
+        }
+
+        # This does not work as expected when runnning in test kitchen and with Chocolatey Licensed Extension installed.
+        # It is believed to be a problem with test kitchen, or the VM that we are using that is causing the issue.
+        # The result is that versioned backup files of each file in the package is created, instead of the package being
+        # removed. Consider the test partially broken.
+        It "Should have removed <_>" -ForEach @('isdependency', 'hasdependency', 'isexactversiondependency') -Tag FossOnly {
+            "$env:ChocolateyInstall\lib\$_" | Should -Not -Exist -Because $Output.String
+        }
+
+        It "Outputs <_> was succcesfully uninstalled" -ForEach @('isdependency', 'hasdependency', 'isexactversiondependency') {
+            $Output.Lines | Should -Contain "$_ has been successfully uninstalled." -Because $Output.String
+        }
+
+        It "Outputs 3/3 packages uninstalled" {
+            $Output.Lines | Should -Contain "Chocolatey uninstalled 3/3 packages." -Because $Output.String
+        }
+    }
+
+    Context "When specifying non-existing package before and after failing package does not abort execution" {
+        BeforeAll {
+            $null = Invoke-Choco install uninstallfailure installpackage

```

Comment:
Do we need a `--confirm` here?

Reply:
Not really, but for conformity, we should.

We do have global confirmation configured already, and I'll get it added (won't be until Tuesday though).

Reply:
@AdmiringWorm I noticed that you have updated this PR, is this request something that you were still planning on adding?

Reply:
I had missed/forgot it. I have updated now to include the `--confirm` flag.